app/router.js

```diff
@@ -33,6 +33,9 @@ const router = Router.extend(RouterScroll, {
 });
 
 router.map(function() {
+  this.route('imprint');
+  this.route('terms-condition');
+  this.route('privacy-policy');
```

Comment:
Why is this route added when there is no page for privacy policy *yet*

Reply:
We need to add `Privacy Policy` but the server went down , So pushed the two pages to check if it is okay